src/google-maps.ts

```diff
@@ -479,14 +526,13 @@ export class GoogleMaps {
                 bounds.extend(markerLatLng);
             }
 
-            this.map.fitBounds(bounds);
-            let listener = google.maps.event.addListener(this.map, 'idle', () => {
-                if (this.map.getZoom() > this.zoom) {
-                    this.map.setZoom(this.zoom);
```

Comment:
We couldn't work out a logical case for this to exist... Zooming out, but not in? It caused some really weird results for us with data that updates...

Reply:
This is legendary. Really nice work @tomtomau @justindra love it. I'll get it released today.